packages/generator-single-spa/src/html/templates/webpack.config.js

```diff
@@ -0,0 +1,51 @@
+const { mergeWithRules } = require("webpack-merge");
+const singleSpaDefaults = require("webpack-config-single-spa");
+
+const mergeRulesByTestMatch = mergeWithRules({
```

Comment:
See this https://github.com/single-spa/create-single-spa/issues/147#issuecomment-705050112 for information.

Reply:
Nice - glad to see webpack-merge being more useful again.